portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +25,28 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String MY_NAME = "Jess DeJong";
+  private ArrayList<String> listOfComments;
```

Comment:
nit: use `List<String>` for the variable type

(always prefer the interface type unless your code assumes a particular implementation)

Reply:
Makes sense, I'll make the change